examples/src/main/java/org/apache/spark/examples/ml/JavaCountVectorizerExample.java

```diff
@@ -54,6 +54,7 @@ public static void main(String[] args) {
       .setOutputCol("feature")
       .setVocabSize(3)
       .setMinDF(2)
+      .setBinary(true)
```

Comment:
I don't think it's really necessary to set each and every possible parameter in every example. I think you can remove the `setBinary` and `setRelativeError` calls from these different examples.